backend/src/repositories/user.repository.ts

```diff
@@ -1,44 +1,36 @@
 import { db } from "../configs/pgdbConnection"
-import { Gender, UserType } from "../models/user/user.model"
+import { UserSchema } from "../models/user/user.model"
 
 export const userRepository = {
   findUser: async (email: string, fName: string, lName: string) => {
-    const user = await db.query("SELECT userid, password FROM user_table WHERE email = $1 OR ( fname = $2 AND lname = $3 )", [email, fName, lName])
-    return user.rows
+    const user = await db.query(
+      "SELECT userid, password, usertype FROM user_table WHERE email = $1 OR ( fname = $2 AND lname = $3 )",
+      [email, fName, lName]
+    )
+    return user.rows[0]
   },
-  createUser: async (
-    fName: string,
-    lName: string,
-    gender: Gender,
-    phoneNumber: string,
-    email: string,
-    birthDate: Date,
-    profilePicture: string,
-    isBanned: boolean,
-    bankName: string,
-    accountNumber: string,
-    password: string,
-    userType: UserType
-  ) => {
+  createUser: async (user: UserSchema) => {
```

Comment:
Next time, please use CreateUserSchema naaa, it gives more sense. But ill approve pai korn